perfkitbenchmarker/benchmark_sets.py

```diff
@@ -125,9 +130,18 @@ def GetBenchmarksFromFlags():
   If multiple sets or mixes of sets and benchmarks are specified, this will
   return the union of all sets and individual benchmarks.
   """
+  # Create a dictionary of valid benchmark names and modules
+  # TODO(voellm): I really think the benchmarks package init should
+  # build the dictionary
+  valid_benchmarks = {}
+  for benchmark_module in benchmarks.BENCHMARKS:
+    valid_benchmarks[benchmark_module.GetInfo()['name']] = benchmark_module
+
   benchmark_names = set()
   for benchmark in FLAGS.benchmarks:
-    if benchmark in BENCHMARK_SETS:
+    if benchmark == ALL_SET:
```

Comment:
Not sure what you mean by "_set is assumed" - as far as I can tell it's just part of the name and I didn't see any special logic that would automatically add or remove a _set suffix. I named this set 'all_set' for consistency with the other sets that also have '_set' suffixes, assuming that names without '_set' suffix are individual benchmarks and not sets. Should I change the name from 'all_set' to just 'all'?